lib/aullar/flair.moon

```diff
@@ -35,8 +35,21 @@ set_line_type_from_flair = (cr, flair) ->
       cr.dash = {6, 3}
 
 draw_ops = {
-  rectangle: (flair, x, y, width, height, cr) ->
+  custom: (flair, x, y, width, height, cr) ->
+    if flair.custom_draw and
+         pcall flair.custom_draw, flair, x, y, width, height, cr
+      return
+    else
+      -- Draw an intensely red "shit has gone bad"-cross instead
```

Comment:
Or just do nothing?

Note that in another failure case - if custom_draw fails - nothing is drawn. (The pcall return value is ignored above), I think in either case logging an error and doing nothing is reasonable.